package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "@expo/samples": "2.1.1",
     "expo": "^32.0.0",
+    "immutable": "^4.0.0-rc.12",
```

Comment:
Just wondering... Shouldn't add of this package to here result in changes to package-lock.json?

Reply:
As of the hack night last evening, it was decided the project would move to yarn from npm, so package-lock.json is no more.